lib/srv/regular/sshserver_test.go

```diff
@@ -452,7 +452,7 @@ func TestAgentForward(t *testing.T) {
 
 	// wait for the output
 	var output []byte
-	for i := 0; i < 100 && len(output) == 0; i++ {
+	for i := 0; i < 400 && len(output) == 0; i++ {
```

Comment:
Nit: I would rewrite this using proper select loop with a timeout on context instead of sleep if you're here anyway but it's up to you.

Reply:
Might be a good candidate for `require.Eventually()`?